azure-pipelines.yml

```diff
@@ -90,7 +90,10 @@ stages:
     ${{ if environment.condition }}:
       condition: ${{ environment.condition }}
     displayName: Build and Test ${{ environment.name }} APIM
-    
+    variables:
+    - group: SonarCloud
```

Comment:
don't need to create another variable group, this can be placed within COMMON group

Reply:
I have added new variable in common group